### DISCUSSION TITLE
Can't delete items with translations

### DISCUSSION THREAD

**Discussion by lluishi93** [directus/directus#13044] [Archive]
### Describe the Bug

A FK constraint is preventing the items to be deleted.

Potentially related to **https://github.com/directus/directus/issues/12478**

### To Reproduce

1. Create collection A
2. Add translations field to collection A
3. Create some items for collection A with its translations
4. Try to delete the item

### Errors Shown

```json
{
  "errors": [
    {
      "message": "delete from `c_cta` where `id` in ('2149') - ER_ROW_IS_REFERENCED_2: Cannot delete or update a parent row: a foreign key constraint fails (`emma_pre_directus9`.`c_cta_translations`, CONSTRAINT `c_cta_translations_entity_foreign` FOREIGN KEY (`entity`) REFERENCES `c_cta` (`id`))",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
```

### What version of Directus are you using?

9.9.1

### What version of Node.js are you using?

16.14.0

### What database are you using?

MySQL 5.7

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
You can configure the relational constraints from the relationship configuration tab of the field:

<img width="966" alt="CleanShot 2022-04-28 at 09 31 21@2x" src="https://user-images.githubusercontent.com/9141017/165763838-01d0a13a-bd7d-493f-8ebc-94cf05841893.png">

In your use case, you might want to set the second one to:

<img width="612" alt="CleanShot 2022-04-28 at 09 34 35@2x" src="https://user-images.githubusercontent.com/9141017/165764343-c8fe613c-e2dc-42cb-9955-b7d2d65b5c74.png">
